clang/include/clang/Tooling/DependencyScanning/DependencyScanningTool.h

```diff
@@ -12,11 +12,13 @@
 #include "clang/Tooling/DependencyScanning/DependencyScanningService.h"
 #include "clang/Tooling/DependencyScanning/DependencyScanningWorker.h"
 #include "clang/Tooling/DependencyScanning/ModuleDepCollector.h"
+#include "clang/Tooling/DependencyScanning/ScanAndUpdateArgs.h"
```

Comment:
This header doesn't provide any types for this header?

Reply:
This is where DepscanPrefixMapping lives.  It was previously forward declared in this file, but that's no longer sufficient.